tests/driver_hd44780/hd44780_params.h

```diff
@@ -0,0 +1,37 @@
+
+#ifndef HD44780_PARAMS_H
+#define HD44780_PARAMS_H
+
+#include "board.h"
+#include "periph/gpio.h"
+
+#include "hd44780.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#define HD44780_PARAMS_ARDUINO {    \
```

Comment:
yeah could do that, however I'm still thinking about howto provide some default pin configs for different (non-arduino boards) in an intuitive fashion. Any suggestions?